Algebras/Shapes/alg_shapes_rope.gap

```diff
@@ -572,27 +298,29 @@ algebra alg_shape2 extends alg_shape5 {
 }
 
 algebra alg_shape1 extends alg_shape5 {
+	include "Algebras/Shapes/Parts/algpart_cofold_shape1_rope.gap"
+
   Rope sadd(Subsequence b, Rope e) {
-    if (front(e) == "_") {
+    if (front(e) == '_') {
```

Comment:
did you compile this? I had some issues with the right hand side being either a string `"_"` or a single char `'_'` 
Or is that something that come from my PR?

Reply:
That one is from your PR

Reply:
Ok, than lets hope I knew what I was doing ;-)